### PR TITLE
Suggestion fixing "ready-up" and "open contracts"

### PR DESCRIPTION
**Pull request by TheBallOfAeolus**

### CODE REVIEW

File: cfg/binds.cfg [hunk 1]
```diff
@@ -87,7 +88,7 @@ bind 	F8 	"record fix; stop; snd_restart; hud_reloadscheme" //This will freese y
 bind	F9	"load_itempreset 0; wait 10; load_itempreset 2; wait 10; load_itempreset 0"
 bind	F10	"load_itempreset 1; wait 10; load_itempreset 2; wait 10; load_itempreset 1"
 bind 	F11	"load_itempreset 2; wait 10; load_itempreset 3; wait 10; load_itempreset 2"
-bind	F12	"load_itempreset 3; wait 10; load_itempreset 2; wait 10; load_itempreset 3"
+bind	F12	 show_quest_log
```
Comment (Tvde1) [inline cfg/binds.cfg]: This key is taken. Perhaps choose another one?

File: cfg/pyro.cfg [hunk 2]
```diff
@@ -28,15 +28,15 @@ alias specialshiftoff "attack2safety; bind mouse2 +airblast"
...
+//alias hideflames "viewmodel_fov 1; r_drawviewmodel 0"
 
 alias pyrovmoff "alias fov hideflames; fovtest"
 alias pyrovmon "alias fov none; normalvmfov"
 
 alias fovteston "alias fovtest fov"
 alias fovtestoff "alias fovtest none"
 
-alias pyrovmtoggle pyrovmtoggle1 //Change this into "pyrovmtoggle2" if you removed hideflames down below.
+alias pyrovmtoggle pyrovmtoggle2 //Change this into "pyrovmtoggle2" if you removed hideflames down below.
```
Comment (Tvde1) [inline cfg/pyro.cfg]: This is personal preference and doesn't need to be committed I think

**Review by Tvde1 — CHANGES_REQUESTED** [review]

File: cfg/binds.cfg (hunk 1 above)
Comment (TheBallOfAeolus) [reply cfg/binds.cfg]: you are right, sorry about that.
I've just updated it and bind the show_quest_log to p

**Comment by TheBallOfAeolus** [comment]
Sorry tvde1, I think I've modified yours.
I am still learning github

File: cfg/pyro.cfg (hunk 2 above)
Comment (TheBallOfAeolus) [reply cfg/pyro.cfg]: You are right, that is my personal preference.
I've just made a brunch with my personal pyro config.

**Comment by TheBallOfAeolus** [comment]
I think that now everything is as it should.

**Comment by Tvde1** [comment]
Looking good 👍

**Pull request merged**